### DISCUSSION TITLE
error about pg_strom.enabled

### DISCUSSION THREAD

**Discussion by naturliche** [heterodb/pg-strom#766] [Questions & Reports]
Hello, I have a problem and don't know how to solve it, and I want to ask. I found that when I get the error **ERROR: parameter "pg_strom.enabled" cannot be changed** when I `set pg_strom.enabled = on`, then I look at the startup log of the pg_ctl and find that there is an error **gpuinfo:459 failed on cuInit: CUDA_ERROR_UNKNOWN**, and I don't know if it is related to it

**Comment by kaigai** [comment]
Does it version 3.x series?
The latest v5.x series don't use `gpuinfo` to collect device properties. Try the latest one.

**Reply by naturliche** [reply]
Thank you for your answer, but I need to use the JIT of NVRTC
